PaddleCV/object_detection/configs/ssd_mobilenet_v1_voc.yml

```diff
@@ -80,7 +80,7 @@ SSDTestFeed:
   batch_size: 1
   dataset:
     dataset_dir: data/voc
-    annotation: VOCdevkit/VOC_all/ImageSets/Main/test.txt
+    annotation: VOCdevkit/VOC_all/ImageSets/Main/label_list.txt
```

Comment:
Why change it?

Reply:
label_list.txt is the label file generate by object_detection/ppdet/data/tools/generate_data_for_training.py, which is use for infer mode

Reply:
For SSD on VOC, better to use default mapping.

Reply:
set use_default_label mapping as true，this has a higher priority